src/shared/components/ActionBar/ActionBar.style.ts

```diff
@@ -17,6 +18,11 @@ export const StyledActionBarContainer = styled.div`
   }
 `
 
+export const StyledInnerContainer = styled(StudioContainer)`
```

Comment:
Not related to this component specifically, but I think we could adjust the breakpoints here a bit:
<img width="1080" alt="Screenshot 2021-03-17 at 15 59 11" src="https://user-images.githubusercontent.com/12646744/111489047-17146e80-873a-11eb-8640-db4f3c2cee78.png">

It seems there's enough space to display the `Every change...` next to the fee instead of below it